k8s/dependencies/opentripplanner/opentripplanner.yaml

```diff
@@ -62,10 +62,10 @@ metadata:
 spec:
   tls:
     - hosts:
-        - otp.iteam.services
+        - otp.predictivemovement.se
       secretName: otp-prod-tls
   rules:
-    - host: otp.iteam.services
+    - host: otp.predictivemovement.se
```

Comment:
For example, we do not keep iteam.services here, but we do for osrm, pelias, opentiles and vroom.